Resources/Prototypes/Entities/Constructible/Specific/Conveyor/conveyor.yml

```diff
@@ -25,34 +25,32 @@
     sprite: Constructible/Power/conveyor.rsi
     state: conveyor_started_cw
     drawdepth: FloorObjects
-
+  - type: SignalReceiver
+    maxTransmitters: 1
   - type: Conveyor
   - type: Appearance
     visuals:
     - type: ConveyorVisualizer
       state_running: conveyor_started_cw
       state_stopped: conveyor_stopped_cw
       state_reversed: conveyor_started_cw_r
-      state_loose: conveyor_loose
   - type: PowerReceiver
+  - type: Construction
+    graph: ConveyorGraph
+    node: entity
 
 - type: entity
-  id: ConveyorSwitch
-  name: conveyor switch
-  description: A conveyor control switch.
+  id: ConveyorBeltItem
```

Comment:
Should this be craftable at some point?

Reply:
yes, by lathe or something

Reply:
yes, lathe i presume?

can i already add that?